src/boot/efi/secure-boot.c

```diff
@@ -126,3 +126,71 @@ EFI_STATUS secure_boot_enroll_at(EFI_FILE *root_dir, const char16_t *path) {
 
         return err;
 }
+
+static EFI_STATUS install_security_override_one(EFI_GUID *guid, SecurityOverride *override) {
```

Comment:
`const` before first arg

Reply:
That creates ugly casting, I've changed it to not take a pointer instead.